crates/ruff/src/checkers/ast/mod.rs

```diff
@@ -745,6 +745,9 @@ where
                     if self.enabled(Rule::EllipsisInNonEmptyClassBody) {
                         flake8_pyi::rules::ellipsis_in_non_empty_class_body(self, stmt, body);
                     }
+                    if self.enabled(Rule::FixedReturnType) {
+                        flake8_pyi::rules::fixed_return_type(self, bases, body, decorator_list);
```

Comment:
Nit: you can reduce the number of arguments by passing the entire class:

1. Change the match arm on line 655 to

```rust
Stmt::ClassDef(class @ ast::StmtClassDef {
                name,
                bases,
                keywords,
                decorator_list,
                body,
                range: _,
}) => {
```

then pass it here 
```suggestion
                        flake8_pyi::rules::fixed_return_type(self, class);
```